modGui/build.gradle

```diff
@@ -26,6 +29,14 @@ dependencies {
     compile files('../lib/libnsc.jar')
     compile 'com.google.guava:guava:25.1-jre'
     compile 'org.slf4j:slf4j-api:1.7.25'
+
+
+    compile files(String.valueOf(System.getenv("JAVAFX_PATH")) + "/lib/javafx.base.jar")
+    compile files(String.valueOf(System.getenv("JAVAFX_PATH")) + "/lib/javafx.controls.jar")
```

Comment:
I remember we were trying to get the JavaFX jars remotely... did it not work?

Reply:
the jar downloaded from MVN repo does not work.